modules/apps/object/object-web/src/main/resources/META-INF/resources/js/utils/normalizeDateInput.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+export function normalizeDateInput(value: string): string {
```

Comment:
This function body is not easy to understand. Its name also is too generic. Please, add a comment in JSDoc format explaining what it does.

Reply:
Also, please add unit test for this function to ensure it really does what you expect it to do and to prevent changes from breaking it. This will also help future developers to understand the expected outputs.

Reply:
I agree! As we aligned, I will work on these changes in the following pr, which will contain the functional tests along with the requested improvements here. thanks!!